src/commands/utils/azureAccount.ts

```diff
@@ -226,5 +225,12 @@ async function getSubscriptionAccess(
 }
 
 function isUnauthorizedError(e: unknown): boolean {
-    return isObject(e) && "code" in e && "statusCode" in e && e.code === "AuthorizationFailed" && e.statusCode === 403;
+    return (
+        typeof e === "object" &&
+        e !== null &&
+        "code" in e &&
+        "statusCode" in e &&
+        e.code === "AuthorizationFailed" &&
+        e.statusCode === 403
```

Comment:
cool, just to add further context for folks who may not be familiar here is the `isObject`  below, so the `constructor` thing motion in details of this PR is referring to the visual difference and contextual built up. thank you.


```*.ts
export function isObject(value: unknown): value is object {
    return typeof value === "object" && value !== null && value.constructor.name === "Object";
}
```